pset-functions.js

```diff
@@ -75,9 +92,38 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
-
+//create function called scrabbleScore with word
+function scrabbleScore(word){
+//convert the word to uppercase and split it to an array
+  const letters = word.toUpperCase().split('')
+//create variable for result scores
+  let score = 0
+//loop through word w/ forEach
+  letters.forEach((letter) => {
```

Comment:
Really great combination of the forEach method with if/else statements; and using the .includes method to run your conditional statements! The iteration updates the sum each turn; very nice job here.